psclient/src/main/java/com/majeur/psclient/service/observer/BattleRoomMessageObserver.kt

```diff
@@ -29,7 +30,11 @@ class BattleRoomMessageObserver(service: ShowdownService)
     var gen = 0
 
     private val battleTextBuilder = BattleTextBuilder(service)
-    private val actionQueue = ActionQueue(Looper.getMainLooper())
+    private val actionQueue = ActionQueue(Looper.getMainLooper(), object : QueueListener {
+        override fun onQueueEmpty() {
```

Comment:
Use onBattleEnded

Reply:
After hooking into 'onBattleEnded' as per your comment elsewhere, it actually made the use of this callback redundant, so I completed removed this.